mamba/mamba/shell_templates/mamba.sh

```diff
@@ -17,7 +17,7 @@ mamba() {
             ;;
         install|update|upgrade|remove|uninstall)
             __mamba_exe "$@" || \return
-            __conda_reactivate
+            __conda_activate reactivate
```

Comment:
Are we sure that `__conda_activate` is always defined?

Reply:
Yes, conda_reactivate was removed because it was duplicating `__conda_activate reactivate`.